tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -3465,3 +3465,371 @@ def create_zeros_like_3_val():
     assert(     f3_shape() ==      create_zeros_like_3_shape()      )
     assert(isclose(     f3_val()  ,      create_zeros_like_3_val()        , rtol=RTOL, atol=ATOL))
     assert(type(f3_val())  == type(create_zeros_like_3_val().item()))
+
+
+@pytest.mark.parametrize( 'language', (
```

Comment:
is there no way to run these test they are skiped in fortran and C

Reply:
i think we can comment int8 data types tests specifically.